controlplane/src/core/controllers/auth.ts

```diff
@@ -171,6 +210,10 @@ const plugin: FastifyPluginCallback<AuthControllerOptions> = function Auth(fasti
     // Set the session cookie. The cookie value is encrypted.
     opts.authUtils.createSessionCookie(res, jwt, sessionExpiresDate);
 
+    if (orgs.length === 0) {
+      res.redirect(opts.webBaseUrl + '?migrate=true');
```

Comment:
Do we have assumed that every user wants to migrate? What will be the impact on the frontend?

Reply:
yep currently, everybody who newly registers will have the dialog opened in the dashboard

Reply:
Got it. Sounds good.